invenio_app_rdm/config.py

```diff
@@ -446,9 +446,15 @@ def files_rest_permission_factory(obj, action):
         "name": _("JSON"),
         "serializer": ("invenio_rdm_records.resources.serializers:"
                        "UIJSONSerializer")
-    }
+    },
+    "datacite": {
+       "name": _("DataCite"),
```

Comment:
```suggestion
       "name": _("DataCite JSON"),
```

Reply:
I wrote DataCite so it would be the same as in Zenodo, should i still change it to DataCite JSON?

Reply:
It's because we'll have both a DataCite XML and DataCite JSON whereas Zenodo only has a DataCite XML.